### DISCUSSION TITLE
4K 10bit to 4K/2K 8bit encoding

### DISCUSSION THREAD

**Discussion by DCNerds** [cdgriffith/FastFlix#549] [Q&A] [answered]
I know.... what kind of idiot would waste time reducing the quality of a movie... so if we are all done throwing popped-corn and making fun of me..... (attempt at humor)

I don't own a 4K 10bit tv yet. Because of that, many 10bit movies look washed out/have little contrast. I have encoded a few movies successfully with the following settings, all but [removed]. I tried to re-encode [removed] with CRF 22, 20 nd 18, with no luck.

Any Ideas? Am I not understanding something? I though I was to use 709 color space for 8bit, and 2020 for 10 bit. Short of buying me a new 4K/10bit monitor, does anybody have any ideas/thought? I am using fastflix 5.6.0 and ffmpeg 6.1.1, screen shots of the setting and the artifacts in the movie to follow.

Thanks for any guidance.

[screenshots removed]

**Comment by cdgriffith** [comment]
Re-adding cleaned images
![setting 2](https://github.com/cdgriffith/FastFlix/assets/3275435/5f496739-8b03-4727-a538-8b6190f826d5)
![setting 1](https://github.com/cdgriffith/FastFlix/assets/3275435/27550848-60bb-4712-9521-16a088034fe5)

**Comment by cdgriffith** [comment, marked as answer]
Hi @DCNerds after some testing with cloud footage, I found out the issue you had was with the selected Tone Map. If set to "hable" those artifacts should disappear.
